@Alyx/newExp.m

```diff
@@ -62,26 +62,22 @@
 
 % Main repository is the reference location for which experiments exist
 [expPath, expRef] = dat.expPath(subject, floor(expDate), expSeq, 'main');
-% ensure a unique `expRef` and `expPath` can be made
-assert(~any(file.exists(expPath)), ...
-  sprintf(['Something went wrong as experiment folders already exist for'...
-  '\n"%s" in \n%s or \n%s. \nIf these folders are empty, delete them and '...
-  'try again.'], expRef, expPath{:}));
-
-% Try making new directories. If this fails, remove empty directories that
-% may have been created, and throw error.
-mkdirFailed.message = sprintf(['Creating experiment directories failed.'...
-    '\nCheck access/credentials for the servers \n%s and \n%s'],... 
-    expPath{:});
-mkdirFailed.identifier = 'Rigbox:alyxmatlab:newExp:mkdirFailed';
-try 
-  all(cellfun(@(p) mkdir(p), expPath)); % make new exp directories
+% ensure nothing went wrong in making a "unique" ref and path to hold
```

Comment:
Nice, this is much better code wise. Small notes: this comment makes it seem like something could go wrong in the actual process of generating `expRef` when in fact we want to see if something goes wrong in creating the directory from `expRef`.

Reply:
As I explain in the troubleshooting guide the ref is generated from folder structure, so they're one in the same thing.  If the ref doesn't reflect a unique underlying folder structure then it's invalid.

Reply:
I meant I don't really like the past tense phrasing (or the scare quote =p ) but not a big deal

Reply:
Sure, this was just part of the revert.  This line is identical the `dat.newExp`, I didn't write it but think it's adequate.  The folder structure is also explained in the docstring.  If I had written it I probably wouldn't have added scare quotes.